src/_variables.scss

```diff
@@ -3,103 +3,222 @@
 
+// WET Custom
 // Additional Viewports outside of Bootstrap definitions
-$screen-xxs-max: 479px !default;
-$screen-lg-max: 1599px !default;
-$screen-xl-min: 1600px !default;
+
+// Extra extra small screen
+$screen-xxs-max:          479px !default;
+// Large screen
+$screen-lg-max:           1599px !default;
+// Extra large screen
+$screen-xl-min:           1600px !default;
+
+
+//== Pagination
+//
+//##
+
+// Bootstrap Overrides
+$pagination-color:        $btn-default-color !default;
+$pagination-bg:           $btn-default-bg !default;
+$pagination-border:       $btn-default-border !default;
+
+$pagination-hover-color:  $btn-default-color !default;
+$pagination-hover-bg:     darken($btn-default-bg, 8%) !default;
+$pagination-hover-border: darken($btn-default-border, 12%) !default;
+
+// WET Custom
+$pager-border-radius:     4px !default;
+
+
+//
+// WET CUSTOM PLUGINS VARIABLES
+//
+
+//== Calendar
+//
+//##
+
+$cal-text-head:           #fff !default;
+$cal-text-body:           #000 !default;
+
+$cal-dark:                #333 !default;
+$cal-medium:              #555 !default;
+
+$cal-active:              #ccc !default;
+$cal-days-link:           #111 !default;
+
+
+//== Footnotes
+//
+//##
+
+$spacing:                 .375em !default;
+$ddRtnWidth:              3.375em !default;
+
+
+//== Multimedia player
+//
+//##
+
+$mm-progress-fg-color:    #176ca7 !default;
+$mm-progress-bg-color:    #fff !default;
+
+$mm-cc-fg-color:          #fff !default;
+$mm-cc-bg-color:          #000 !default;
+
+$mm-ctrl-fg-color:        #fff !default;
+$mm-ctrl-bg-color:        #3e3e3e !default;
+$mm-ctrl-outline-color:   #4aafff !default;
+$mm-ctrl-cc-color:        #4aafff !default;
+
+$mm-volume-bg-color:      #aaa !default;
+$mm-volume-thumb-color:   #fff !default;
+
+
+//== Tabs
+//
+//##
+
+// Tabbed Interface
+$tablist-color:                  #000 !default;
+$tablist-border:                 $clrMedium !default;
+$tabpanel-border:                $tablist-border !default;
+$tablist-bg:                     #ebf2fc !default;
+$tablist-hover-bg:               $clrMedium !default;
+$tablist-active-bg:              #fff !default;
+$tablist-active-link-border-top: #666 !default;
+// Tablist at sm max && xxs max
+$tablist-screen-sm-details-border:           $tablist-border !default;
+$tablist-screen-sm-details-border-radius:    4px !default;
+
+
+// Carousel
+$carousel-tablist-tabcount-color:            #000 !default;
+
+$carousel-tabpanel-figcaption-color:         #fff !default;
+$carousel-tabpanel-figcaption-link-color:    $carousel-tabpanel-figcaption-color !default;
+$carousel-tabpanel-figcaption-bg:            $clrDrabBlue !default;
+
+// Carousel Style 2 with Thumbnails
+$carousel-s2-tablist-controls-bg:            $clrLight !default;
+
+$carousel-s2-tablist-controls-color:         $clrDrabBlue !default;
+$carousel-s2-tablist-controls-button-bg:     #fff !default;
+$carousel-s2-tablist-controls-button-shadow: $clrDrabBlue !default;
```

Comment:
I am wondering if this should be changed to the whole shadow so that you can turn it off completely if you want

```
$carousel-s2-tablist-controls-button-shadow: 0 0 4px $clrDrabBlue !default;
```